homeassistant/components/elmax/config_flow.py

```diff
@@ -78,55 +74,57 @@ async def async_step_user(
             client = Elmax(username=username, password=password)
             await client.login()
 
-            # If the login succeeded, retrieve the list of available panels and filter the online ones
-            online_panels = [x for x in await client.list_control_panels() if x.online]
-
-            # If no online panel was found, we display an error in the next UI.
-            panels = list(online_panels)
-            if len(panels) < 1:
-                raise NoOnlinePanelsError()
-
-            # Show the panel selection.
-            # We want the user to choose the panel using the associated name, we set up a mapping
-            # dictionary to handle that case.
-            panel_names: dict[str, str] = {}
-            username = client.get_authenticated_username()
-            for panel in panels:
-                _store_panel_by_name(
-                    panel=panel, username=username, panel_names=panel_names
-                )
-
-            self._client = client
-            self._panel_names = panel_names
-            schema = vol.Schema(
-                {
-                    vol.Required(CONF_ELMAX_PANEL_NAME): vol.In(
-                        self._panel_names.keys()
-                    ),
-                    vol.Required(CONF_ELMAX_PANEL_PIN, default="000000"): str,
-                }
+        except ElmaxBadLoginError:
+            _LOGGER.error("Wrong credentials or failed login")
```

Comment:
Do we need to log the error in the logs? We already provide feedback in the UI?

Reply:
Not necessarily, but in my opinion errors should always go to log and - if user-friendly - then go to UI. In this case UI feedback is proveded thanks to the errors dictionary. I believe this also can be a matter of style. Jus let me know if you guys prefer saving log entries when errors are UI-displayed and I'll remove this line.

Reply:
We should not log this. Please remote it.

Reply:
Alright, done.